code/modules/events/drone_pod_vr.dm

```diff
@@ -22,7 +22,7 @@
 		qdel(target_spot)
 
 /datum/event/drone_pod_drop/announce()
-	command_announcement.Announce("An unidentified drone pod has been detected on a collision course towards the [location_name()]. Open and examine at your own risk.", "[location_name()] Sensor Network")
+	command_announcement.Announce("An unidentified drone pod has been detected on a collision course towards [location_name()]. Open and examine at your own risk.", "[location_name()] Sensor Network")
```

Comment:
```suggestion
	command_announcement.Announce("An unidentified drone pod has been detected on a collision course towards [location_name()]. Open and examine at your own risk.", "[location_name()] Sensor Network") // YW Edit Fixed typo
```
It's important to make comments when editing stock files even when fixing a typo.